requirements_dev.txt

```diff
@@ -7,4 +7,6 @@ coverage
 pytest
 pytest-runner
 pytest-mock
+ipython
```

Comment:
Why is `ipython` required? I do not see it used in this PR.

Reply:
`ipython` is a better Python REPL that I used to validate information while writing code. I left it in the requirements so other people can utilize it for debugging purposes.